src/main/scala/bio/ferlab/ferload/endpoints/LegacyObjectEndpoints.scala

```diff
@@ -0,0 +1,70 @@
+package bio.ferlab.ferload.endpoints
+
+import bio.ferlab.ferload.Config
+import bio.ferlab.ferload.model.{ErrorResponse, ObjectUrl, User}
+import bio.ferlab.ferload.services.{AuthorizationService, S3Service}
+import cats.effect.IO
+import cats.implicits.*
+import io.circe.generic.auto.*
+import sttp.model.StatusCode
+import sttp.tapir.*
+import sttp.tapir.generic.auto.*
+import sttp.tapir.json.circe.*
+import sttp.tapir.server.*
+
+object LegacyObjectEndpoints:
+
+
+  private def securedGlobalEndpoint(authorizationService: AuthorizationService, resourceGlobalName: String): PartialServerEndpoint[String, User, Unit, (StatusCode, ErrorResponse), Unit, Any, IO] =
+    endpoint
+      .securityIn(auth.bearer[String]())
+      .errorOut(statusCode.and(jsonBody[ErrorResponse]))
+      .serverSecurityLogic(token => authorizationService.authLogic(token, Seq(resourceGlobalName)))
+
+  private def objectByPath(authorizationService: AuthorizationService, resourceGlobalName: String): PartialServerEndpoint[String, User, List[String], (StatusCode, ErrorResponse), ObjectUrl, Any, IO] =
+    securedGlobalEndpoint(authorizationService, resourceGlobalName)
+      .get
+      .description("Retrieve an object by its path and return an url to download it")
+      .deprecated()
+      .in(paths.description("Path of the object to retrieve"))
+      .out(jsonBody[ObjectUrl])
+
+  private def objectsByPaths(authorizationService: AuthorizationService, resourceGlobalName: String): PartialServerEndpoint[String, User, String, (StatusCode, ErrorResponse), Map[String, String], Any, IO] =
+    securedGlobalEndpoint(authorizationService, resourceGlobalName)
+      .description("Retrieve a list of objects by their paths and return a list of download URLs for each object")
+      .deprecated()
+      .post
+      .in("downloadLinks")
+      .in(stringBody.description("List of URLs of objects to retrieve").example("file1.vcf\nfile2.vcf"))
+      .out(jsonBody[Map[String, String]]
+        .description("List of files URLs by object path")
+        .example(Map("file1.vcf" -> "https://file1.vcf", "file2.vcf" -> "https://file2.vcf"))
+      )
+
+  def objectByPathServer(authorizationService: AuthorizationService, s3Service: S3Service, resourceGlobalName: String, defaultBucket: String): ServerEndpoint[Any, IO] =
+    objectByPath(authorizationService, resourceGlobalName).serverLogicSuccess { user =>
+      file => s3Service.presignedUrl(defaultBucket, file.mkString("/")).pure[IO].map(ObjectUrl.apply)
```

Comment:
current response could contains double `//` if the file is a path ex with clin blue QA
`    "/blue/0f86d5ad-e6a5-4142-ba1f-b254bc61275f.tbi": "http://127.0.0.1:9000/bucket//blue/0f86d5ad-e6a5-4142-ba1f-b254bc61275f.tbi?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20231010T190827Z&X-Amz-SignedHeaders=host&X-Amz-Expires=3600&`
=> **/bucket//blue/** <=
Note: doesnt seems to be problematic but just in case ...